projects/web/src/app/inicio/inicio.component.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../tema";
+
+.centrar {
+  text-align: center;
+}
+main {
+  height: calc(100% - 56px);
+
+  section {
+    margin: 16px 28px;
+    max-width: 800px;
+
+    @media (max-width: 350px) {
+      &#llamada-a-la-acción {
```

Comment:
Maybe code should still probably be in English. These could be confusion for other people trying to collaborate.

Reply:
Thank you, that's really helpful feedback! 

I know that some projects in Perú are done in all English, but those are usually for foreign companies. I know that some projects are done with complete non-English speakers, but I haven't seen what that code looks like.

P.s. I don't anticipate non-Spanish speakers contributing to this project, but it would still make it harder to get help from an expert on a specific problem.

Reply:
Writing your code and your variables/classes in the same language helps consistency. It's one of the first things we have to correct in Spain when training junior developers.
There are other languages in Spain just as Basque, Galicia or Catalan. There are codebases where the naming might be 2 languages + English...

Reply:
Yeah, I can see how that could be chaotic. This is helpful to know as we'll want to put together a Contributors guide at some point and this would be a key point. I guess that only user-visible content should actually be in Spanish, the rest in English.

In Latin America, there are a lot of other languages too (Quechua, Aymara, Mayan, etc).